TranslationValidationPlugin.cs

```diff
@@ -65,6 +66,11 @@ public void Run(IHost host, string activeProjectName)
                     {
                         try
                         {
+                            //CultureInfo englishUSCulture = new CultureInfo("en-US");
```

Comment:
some reason to leave this in? if so, a comment would be helpful.

Reply:
Nope. Removed.